### ISSUE TITLE
Showing the recording area rectangle breaks the mouse clicks

### ISSUE THREAD

**Issue by nekohayo** [iacopodeenosee/EasyScreenCast#24]
If I enable the option "Show area screencast during recording" (by the way it would be better to name this option "Show a border around the area being recorded"), I am not able to click (or drag) as long as the screencast recording is going on; I can't even stop the recording, unless I Alt+F2 & "r" to reload gnome-shell 3.16.

**Comment by iacopodeenosee** [comment]
i think this ia a dupicate of #7 , if you use the latest version of extension can you send me the log? i'm on arch with GS 3.16.2 and this problem is not present.

**Comment by nekohayo** [comment]
Yes, I'm using the latest version of the extension... discovered something weird though: the problem occurs for certain types of actions and certain types of windows only. The best way to reproduce the issue is to try to use Pitivi.

* The "Welcome" dialog on startup cannot be dragged by its titlebar if under an area highlighted by easyscreencast
* The Super+drag key/mouse combo doesn't work in any area where easyscreencast is highlighting; but as soon as you drag a window outside that area, Super+drag works.

**Comment by nekohayo** [comment]
...and I don't know how to give you a proper debug log.

**Comment by iacopodeenosee** [comment]
i see this bug on Pitivi program, and it is very strange, the button on "Welcome" dialog works perfectly into the area recording but the windows decoration don't...I did not find anything useful in the logs, I'll try to look into the sources of pitivi to understand that windows are

**Comment by nekohayo** [comment]
AFAIK there isn't really something special with that dialog in Pitivi, it is modal but that's otherwise normal. The relevant code is:

https://git.gnome.org/browse/pitivi/tree/pitivi/application.py#n172

```
        self.createMainWindow()
        self.welcome_wizard = StartUpWizard(self)
        self.welcome_wizard.show()
```

https://git.gnome.org/browse/pitivi/tree/pitivi/dialogs/startupwizard.py

```
    def show(self):
        """Will show the interal window and position the wizard"""
        self.window.set_transient_for(self.app.gui)
        self.window.set_position(Gtk.WindowPosition.CENTER_ON_PARENT)
        self.window.show()
```

https://git.gnome.org/browse/pitivi/tree/data/ui/startupwizard.ui

```
  <object class="GtkWindow" id="window1">
    <property name="can_focus">False</property>
    <property name="title" translatable="yes">Welcome</property>
    <property name="resizable">False</property>
    <property name="modal">True</property>
    <property name="window_position">center-on-parent</property>
    <property name="destroy_with_parent">True</property>
    <property name="skip_taskbar_hint">True</property>
```

**Comment by massimiliano-cannata** [comment]
I confirm, same bug with my laptop. 
Ubuntu 18.04, 
EasyScreencast Version: 38, 
Gnome: 3.28.2, 
graphic Intel® UHD Graphics 620 (Kabylake GT2),  
processor: Intel® Core™ i7-8550U CPU @ 1.80GHz × 8. 
Using Wayland !

Thanks, great tool !!!

**Comment by julienw** [comment]
I found the same issue. Curiously this doesn't happen when I use the option "capture a specific window" and that window is full screen.

EasyScreencast Version: 40
Gnome: 3.30 on Debian Buster (current stable)
Using Wayland

Thanks for this very useful tool!

**Comment by adangel** [comment]
Note: the option naming ("Show area screencast during recording" -> "Show a border around the area being recorded") has been fixed a while ago in 4689694a290fce8bb518d4a144b05ab827a94ae3.
The option is named here in the current code:
https://github.com/EasyScreenCast/EasyScreenCast/blob/a2dcc6fa44bcf11472aab6dfe7314ecd235330d9/Options_UI.glade#L67
